scripts/zones/Halvung/npcs/_1qp.lua

```diff
@@ -2,35 +2,26 @@
 -- Area: Halvung
 --  NPC: Decorative Bronze Gate (_1qp)
 -----------------------------------
-local ID = require("scripts/zones/Halvung/IDs")
-require("scripts/globals/npc_util")
------------------------------------
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
-    if
-        npcUtil.tradeHasExactly(trade,
-        {
-            xi.items.HALVUNG_SHAKUDO_KEY,
-            xi.items.HALVUNG_BRONZE_KEY,
-            xi.items.HALVUNG_BRASS_KEY
-        })
-    then
+    local item = xi.items.HALVUNG_SHAKUDO_KEY
+
+    if npcUtil.tradeHasExactly(trade, { item, item + 1, item + 2 }) then
         player:confirmTrade()
-        npc:openDoor()
-        player:messageSpecial(ID.text.KEY_BREAKS,
-            xi.items.HALVUNG_SHAKUDO_KEY,
-            xi.items.HALVUNG_BRONZE_KEY,
-            xi.items.HALVUNG_BRASS_KEY
```

Comment:
Are these in the DB to drop? At one point I recall putting them in LSB but not sure if they trickled their way here yet?

Reply:
yup! They seem to all drop at a 5% droprate from various mobs in zone

Reply:
Nice